scipy-stubs/sparse/linalg/_eigen/arpack/arpack.pyi

```diff
@@ -5,15 +5,23 @@ import numpy as np
 import optype.numpy as onp
 import optype.numpy.compat as npc
 
-from scipy.sparse._base import _spbase
+from scipy.sparse._base import _SpArray2D, _SpMatrix
```

Comment:
these private type aliases might change, so let's use the `spmatrix[T]` and `sparray[T, tuple[int, int]]` directly